### PR TITLE
Update latest release section for 0.41.0 in website

### PR DESCRIPTION
**Pull request by Sreekala-Gopakumar**
https://github.com/eclipse-openj9/openj9-website/issues/358

Update latest release section for 0.41.0 in website

Closes #358
Signed-off-by: Sreekala Gopakumar <sreekala.gopakumar@ibm.com>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -118,18 +118,25 @@ <h2>Join community calls</h2>
...
+<p>We're pleased to announce the availability of Eclipse OpenJ9™ v0.41.0.</p>
+<p>This release supports OpenJDK version 8, 11, and 17. For more information about supported platforms and OpenJDK versions, see <a class="w3-hover-opacity" href="https://www.eclipse.org/openj9/docs/openj9_support/" target="_blank">Supported environments</a>.</p>
 <p>Other updates in this release include the following:</p>
 <ul>
-  <li>Support for persistent shared classes cache is now added on z/OS&reg; systems.</li>
-  <li>A new <code>-XX:codecachetotalMaxRAMPercentage</code> option is added to set the maximum limit for the total JIT code cache size based on the amount of physical memory available to the VM.</li>
-  <li>Even though the container detection fails when you mount the <code>/proc</code> file system with the <code>hidepid=2</code> option on Linux® platforms, the VM starts but with a warning message.</li>
+  <li>The `-XX:+CompactStrings` option is now enabled by default on Java&trade; 11 and later.</li>
```
Comment (pshipton) [inline index.html:128]: I don't think the backquotes work in html code.

Comment (pshipton) [reply index.html:128]: I mean, you don't get any formatting change, you just see the backquotes on the web page.

File: index.html [hunk 2]
```diff
@@ -118,18 +118,25 @@ <h2>Join community calls</h2>
...
+  <li>The JITServer client can now use the JITServer AOT cache even if both the `-Xshareclasses:readonly` option and the JITServer AOT cache feature were enabled at the same time. With the change in behavior of the `-Xshareclasses:readonly` option, the shared class cache startup creates a temporary new (writable) top layer that the JITServer AOT cache can use to store data that it needs to function.</li>
+  <li>A new `-XX:[+|-]EnableDynamicAgentLoading` option is added to enable or disable the dynamic loading of agents into a running VM.</li>
+  <li>A new `-XX:[+|-]UseZlibNX` option is added to control the loading of the `zlibnx` library in the `LIBPATH` environment variable on the AIX systems.</li>
```
Comment (pshipton) [inline index.html:131]: "on the AIX systems" could be "on AIX systems"

File: index.html [hunk 3]
```diff
@@ -118,18 +118,25 @@ <h2>Join community calls</h2>
...
 </ul>   
 <p>To read more about these and other changes, see the <a class="w3-hover-opacity" href="https://www.eclipse.org/openj9/docs/openj9_releases/" target="_blank">OpenJ9 user documentation</a>.</p>
+<p><b>Performance highlights include:</b></p>
+<ul>
+  <li>Performance of algorithms is enhanced with the use of OpenSSL version 3 and later. Improved algorithms include SHA256, AES, HmacSHA256, and CHACHA20.</li>
+  <li>For Java 17, the performance to call the virtual or interface methods through `MethodHandles` is improved.</li>
```
Comment (pshipton) [inline index.html:137]: I would change "to call the virtual or interface methods" to "to call virtual or interface methods"

**Comment by Sreekala-Gopakumar** [comment]
@pshipton - Made the changes. Thanks!

**Comment by Sreekala-Gopakumar** [comment]
@pshipton - I have updated the page. Please check and confirm. Thanks!

**Comment by pshipton** [comment]
I think we can mention both "3.0.12, 3.1.4" since both these versions contain the fix. If we just mention 3.0.12, 3.1.0 may be considered a "later" version.

**Comment by Sreekala-Gopakumar** [comment]
> I think we can mention both "3.0.12, 3.1.4" since both these versions contain the fix. If we just mention 3.0.12, then 3.1.0 may be considered a "later" version.

You mean instead of saying "... use OpenSSL version 3.0.12 or later... ", we should say "... use OpenSSL version 3.0.12 or 3.1.4..."?

**Comment by pshipton** [comment]
Yes

**Comment by Sreekala-Gopakumar** [comment]
Done

**Comment by pshipton** [comment]
You removed "or later", which still applies. 3.0.12 or later, 3.1.3 or later.

**Comment by Sreekala-Gopakumar** [comment]
> You removed "or later", which still applies. 3.0.12 or later, 3.1.4 or later.

This might be confusing. Which version is not applicable?

**Comment by pshipton** [comment]
Discussed on Slack and suggested 3.0.12 or a later 3.0.x version, 3.1.4 or a later 3.1.x version.

The point being that either 3.0.x or 3.1.x can be used, but the required fix is introduced in 3.0.12 and 3.1.4.

**Comment by Sreekala-Gopakumar** [comment]
Incorporated the suggested change. Please check and confirm. Thanks!

**Comment by Sreekala-Gopakumar** [comment]
Updated the content based on the feedback received in https://github.com/eclipse-openj9/openj9-docs/pull/1200#issuecomment-1784392639

**Comment by Sreekala-Gopakumar** [comment]
@pshipton - I have updated the release date. Added content related to com.sun.management.ThreadMXBean.getThreadAllocatedBytes() API support back ported from 0.42.release. Please check and confirm. Thanks!

File: index.html [hunk 4]
```diff
@@ -118,18 +118,26 @@ <h2>Join community calls</h2>
...
+  <li>The JITServer client can now use the JITServer AOT cache even if both the <code>-Xshareclasses:readonly</code> option and the JITServer AOT cache feature were enabled at the same time. With the change in behavior of the <code>-Xshareclasses:readonly</code> option, the shared class cache startup creates a temporary new (writable) top layer that the JITServer AOT cache can use to store data that it needs to function.</li>
+  <li>A new <code>-XX:[+|-]EnableDynamicAgentLoading</code> option is added to enable or disable the dynamic loading of agents into a running VM.</li>
+  <li>A new <code>-XX:[+|-]UseZlibNX</code> option is added to control the loading of the <code>zlibnx</code> library in the <code>LIBPATH</code> environment variable on AIX&reg; systems.</li>
+  <li>Support is added for the <code>com.sun.management.ThreadMXBean.getThreadAllocatedBytes()</code> API on all platforms except z/OS&reg;. The <code>isThreadAllocatedMemorySupported()</code>, <code>setThreadAllocatedMemoryEnabled()</code>, and <code>isThreadAllocatedMemoryEnabled()</code> methods are supported now.</li>
```
Comment (pshipton) [inline index.html:132]: See my comment on the release notes about the methods being "supported now".

Comment (pshipton) [reply index.html:132]: Pls indicate the z/OS support will be in the next release.

**Comment by Sreekala-Gopakumar** [comment]
@pshipton - I have made the changes. Please check and confirm.

I have removed the details to maintain the consistency with the other points. Bullet point would have been too big. Anyways, we are mentioning in the next line  that "To read more about these and other changes, see the [OpenJ9 user documentation](https://www.eclipse.org/openj9/docs/openj9_releases/)." I hope that is ok. 

Thanks!

**Comment by pshipton** [comment]
lgtm

**Review by pshipton — APPROVED** [review]

**Pull request merged**